apiary-metastore-listener/src/test/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListenerTest.java

```diff
@@ -135,6 +138,8 @@ public void onInsert() throws MetaException {
         + PROTOCOL_VERSION
         + "\",\"eventType\":\"INSERT\",\"dbName\":\"some_db\",\"tableName\":\"some_table\",\"files\":[\"file:/a/b.txt\",\"file:/a/c.txt\"],"
         + "\"fileChecksums\":[\"123\",\"456\"],\"partitionKeyValues\":{\"load_date\":\"2013-03-24\",\"variant_code\":\"EN\"}}"));
+
+    verifyMessageAttributes(publishRequest,  "INSERT");
```

Comment:
Minor, but you could consider using the enum here and calling toString() on it in the verify method, at least that removes a few hardcoded strings and gets you proper type checking and refactoring support if we ever decided to change the values in the enum in future.